beets/autotag/mb.py

```diff
@@ -57,6 +57,7 @@ def get_message(self):
             self._reasonstr(), self.verb, repr(self.query)
         )
 
+
```

Comment:
Looks like a stray newline snuck in here?

Reply:
done